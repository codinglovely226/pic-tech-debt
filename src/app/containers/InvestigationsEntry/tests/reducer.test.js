/* eslint-disable */

import InvestigationsReducer from '../reducer';

import {
  updateInvestigationsRequests,
  updateInvestigationsResults,
} from '../actions/case';

describe('InvestigationsReducer', () => {
  it('returns the initial state', () => {
    expect(InvestigationsReducer(undefined, {})).toMatchSnapshot();
  });
  it('update requests', () => {
    expect(InvestigationsReducer([], updateInvestigationsRequests(1, 0, 1))).toMatchSnapshot();
  });
  it('update results', () => {
    expect(InvestigationsReducer([{'results': []}], updateInvestigationsResults(1, 0, 4))).toMatchSnapshot();
  });
});
