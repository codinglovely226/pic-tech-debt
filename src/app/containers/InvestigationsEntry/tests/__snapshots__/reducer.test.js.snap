exports[`InvestigationsReducer returns the initial state 1`] = `Array []`;

exports[`InvestigationsReducer update requests 1`] = `
Array [
  Object {
    "request": 1,
    "results": Array [],
  },
]
`;

exports[`InvestigationsReducer update results 1`] = `
Array [
  Object {
    "results": Array [
      4,
    ],
  },
]
`;
